params.go

```diff
@@ -0,0 +1,17 @@
+package dnscache
+
+type Param struct {
```

Comment:
As mentioned previously, these are idiomatically called `Option`.

```suggestion
type Option struct {
```